Sources/DeepLearning/Optimizer.swift

```diff
@@ -39,27 +37,34 @@ public protocol Optimizer {
 ///
 /// Reference: ["Adam - A Method for Stochastic Optimization"](
 /// https://arxiv.org/abs/1412.6980v8)
-public class Adam<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
+public class Adam<Model: Layer>: Optimizer
     where Model.AllDifferentiableVariables == Model.CotangentVector {
     /// The learning rate.
-    public var learningRate: Scalar
+    public var learningRate: Float
     /// A coefficient used to calculate the first and second moments of
     /// gradients.
-    public var beta1: Scalar
+    public var beta1: Float
     /// A coefficient used to calculate the first and second moments of
     /// gradients.
-    public var beta2: Scalar
-    /// A small scalar added to the denominator to improve numerical stability.
-    public var epsilon: Scalar
+    public var beta2: Float
+    /// A small Float added to the denominator to improve numerical stability.
```

Comment:
Same. The comment doesn't need to be changed.

Reply:
Fixed.